src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -1383,11 +1417,27 @@ public NClob getNClob(String parameterName) throws SQLException {
 
     /* L3 */ public java.sql.Array getArray(String sCol) throws SQLServerException {
         checkClosed();
-        return getArray(findColumn(sCol));
+        return getArray(findColumn(sCol, false));
+    }
+    
+    /**
+     * handle `@name` as well as `name`, since `@name` is what's returned by DatabaseMetaData#getProcedureColumns
+     * 
+     * @param columnName
+     * @return
+     */
+    private String concatAtSign(String columnName) {
```

Comment:
This doesn't concatenate an "@" to anything, it removes one (if present), so perhaps it should be called `stripLeadingAtSign`.

Reply:
@gordthompson That's right. Changed it to the suggested name. Thanks!